android/src/main/kotlin/com/onegini/mobile/sdk/flutter/OneWelcomeWrapperErrors.kt

```diff
@@ -2,24 +2,36 @@ package com.onegini.mobile.sdk.flutter
 
 enum class OneWelcomeWrapperErrors(val code: Int, val message: String) {
   GENERIC_ERROR(8000, "Something went wrong"),
-  USER_PROFILE_DOES_NOT_EXIST(8001, "The requested User profile does not exist"),
-  NO_USER_PROFILE_IS_AUTHENTICATED(8002, "There is currently no User Profile authenticated"),
-  AUTHENTICATOR_NOT_FOUND(8004, "The requested authenticator is not found"),
-  HTTP_REQUEST_ERROR(8011, "OneWelcome: HTTP Request failed internally"),
-  ERROR_CODE_HTTP_REQUEST(8013, "OneWelcome: HTTP Request returned an error code. Check Response for more info"),
+  NOT_AUTHENTICATED_USER(8002, "There is currently no User Profile authenticated"),
+  NOT_AUTHENTICATED_IMPLICIT(8002, "The requested action requires you to be authenticated implicitly"),
+  NOT_FOUND_USER_PROFILE(8004, "The requested User profile is not found"),
+  NOT_FOUND_AUTHENTICATOR(8004, "The requested authenticator is not found"),
+  NOT_FOUND_IDENTITY_PROVIDER(8004, "The requested identity provider is not found"), // Android only
+  NOT_FOUND_SECURITY_CONTROLLER(8004, "The requested Security controller class is not found"), // Android only
+  HTTP_REQUEST_ERROR_INTERNAL(8011, "OneWelcome: HTTP Request failed internally"),
+  HTTP_REQUEST_ERROR_CODE(8011, "OneWelcome: HTTP Request returned an error code. Check Response for more info"),
```

Comment:
Message: "OneWelcome: HTTP Request returned an http error code. Check Response for more info"),

Reply:
done and slightly further updatwed